src/Security/Request/RequestVerifierInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Security\Request;
+
+use Symfony\Component\HttpFoundation\Request;
+
+interface RequestVerifierInterface
+{
+    public function verifyRequest(Request $request, string $publicKey): bool;
```

Comment:
If method returns bool, maybe better to name it somthing like `isValidRequest()`?

Reply:
here the meaning is in the verification and not in the validation